apps/admin-gui-e2e/src/integration/facility-management-perun_admin.spec.js

```diff
@@ -0,0 +1,212 @@
+      .get('[data-cy=delete-facility-button]')
+      .click()
+      .get('[data-cy=delete-button-dialog]')
+      .click()
+      .wait('@deleteFacility')
+      .wait('@getEnrichedFacilities')
+      .get(`[data-cy=${dbFacilityName1}-checkbox]`)
+      .should('not.exist')
+  });
+
+  it('test create resource', () => {
+
+    cy.intercept('**/facilitiesManager/getEnrichedFacilities').as('getEnrichedFacilities')
+      .intercept('**/vosManager/getAllVos').as('getAllVos')
+      .intercept('**/facilitiesManager/getAssignedRichResources?**').as('getAssignedResources')
+      .intercept('**/resourcesManager/createResource?**').as('createResource')
+
+      .wait('@getEnrichedFacilities')
+      .get(`[data-cy=${dbFacilityName2}]`)
+      .click()
+      .get('[data-cy=resources]')
+      .click()
+      .get('[data-cy=create-resource-button]')
+      .click()
+      .wait('@getAllVos')
+      .get('[data-cy=create-resource-select-vo]')
+      .click()
+      .get('mat-option').contains(`${dbVoName}`).click()
+      //.get(`[data-cy=${dbVoName}]`)
+      //.click()
+      .get('[data-cy=create-resource-name-input]')
+      .type('test-e2e-resource')
+      .get('[data-cy=create-resource-dialog-button]')
+      .click()
+      .wait('@createResource')
+      .wait('@getAssignedResources')
+      .get('[data-cy=test-e2e-resource-checkbox]')
+      .should('exist')
+  });
+
+  it('test delete resource', () => {
+
+    cy.intercept('**/facilitiesManager/getEnrichedFacilities').as('getEnrichedFacilities')
+      .intercept('**/facilitiesManager/getAssignedRichResources?**').as('getAssignedResources')
+      .intercept('**/resourcesManager/deleteResource?**').as('deleteResource')
+
+      .wait('@getEnrichedFacilities')
+      .get(`[data-cy=${dbFacilityName2}]`)
+      .click()
+      .get('[data-cy=resources]')
+      .click()
+      .get(`[data-cy=${dbResourceName}-checkbox]`)
+      .click()
+      .get('[data-cy=delete-resource-button]')
+      .click()
+      .get('[data-cy=delete-button-dialog]')
+      .click()
+      .wait('@deleteResource')
+      .wait('@getAssignedResources')
+      .get(`[data-cy=${dbResourceName}-checkbox]`)
+      .should('not.exist')
+  });
+
+  it('test add attribute', () => {
+
+    cy.intercept('**/attributesManager/setAttributes/**').as('setAttributes')
+      .intercept('**/facilitiesManager/getEnrichedFacilities').as('getEnrichedFacilities')
+      .wait('@getEnrichedFacilities')
+      .get(`[data-cy=${dbFacilityName2}]`)
+      .click()
+      .get('[data-cy=attributes]')
+      .click()
+      .get('[data-cy=add-attributes]')
+      .click()
+      .get('[data-cy=filter-attributes]')
+      .type('Login namespace')
+      .get(`[data-cy=${addedAttribute}-value]`)
+      .type('einfra')
+      .get('[data-cy=save-selected-attributes]')
+      .click()
+      .wait('@setAttributes')
+      // assert that attribute exists
+      .get(`[data-cy=${addedAttribute}-value]`)
+      .should('exist')
+  });
```

Comment:
In this test (test add attribute) the final `.should('exist')` checks the wrong HTML element. It checks the element inside the dialog due to a quick check before closing this dialog. 
![Screenshot from 2022-03-01 21-09-37](https://user-images.githubusercontent.com/70371638/156246113-2cfe253b-828d-4f2f-b7bb-4191a4f5953e.png)

I am not sure if you will be able to intercept some calls to the backend which would fix it so you can add some check for example to the header of the dialog and check that it `should('not.exist')`.
![Screenshot from 2022-03-01 21-35-02](https://user-images.githubusercontent.com/70371638/156246153-015e157a-2ef2-4375-8480-e2496da49b25.png)